src/components/Stars/index.js

```diff
@@ -47,7 +47,10 @@ class Stars extends PureComponent<{
   onDragEnd = ({ source, destination }) => {
     const { dragDropStarAction, starredAccounts } = this.props
     if (destination) {
-      dragDropStarAction({ from: source.index, to: destination.index, starredAccounts })
```

Comment:
I was providing the list of accounts here because in the chance that we deleted an account, then the indexes no longer match, I'm not sure how this would behave

Reply:
As an attempt to prevent that issue, and to reliably move the correct account in the complete list when the drag and drop is done while empty accounts are hidden, I'm no longer sending indexes but account ids for `from` and `to`. I should probably have changed the object keys to something like `fromId` and `toId` to make it obvious though...

Reply:
👏